namui-prebuilt/src/typography/mod.rs

```diff
@@ -0,0 +1,47 @@
+pub mod body;
```

Comment:
I think I saw this `typography things` in the previous pr. Can you check it out?